databases/postgresql17/Portfile

```diff
@@ -41,6 +41,8 @@ use_bzip2           yes
 platform darwin powerpc {
     patchfiles-append  patch-icu.diff \
                        patch-fix-ppc-asm.diff
+    depends_build-append \
+                        path:bin/perl:perl5
```

Comment:
Sorry, I should have mentioned this explicitly: for v17 perl5 is not specific to powerpc. It is already among dependencies, so instead of adding it inside ppc block just fix the existing spec.

Reply:
changed ... sorry, force-push was denied for some reason ...

Reply:
Thank you